src/wam2layers/cli.py

```diff
@@ -164,7 +164,7 @@ def era5(ctx, config_file):
     _copy_config_yaml(config_file, log_path)
     logger.info("Welcome to WAM2layers.")
     logger.info("Starting preprocessing ERA5 data.")
-    prep_experiment(config_file)
+    prep_experiment(config_file, data_source="ERA5")
```

Comment:
I like this syntax. Could we eventually connect this to the command line argument as well? Now we have "wam2layers preprocess era5 ...". It could either get the options from supported data sources, or we could modify the config file to include data source, and get rid of this CLI option altogether.